chall10/estina.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+import sys
+
+ROWS = 6
+COLS = 7
+
+def check_line(line, colors, split_line):
+	if len(split_line[0]) != 1:
+		exit("Invalid input.")
+	if not len(colors) or (len(colors) != 2 and colors[0] != split_line[1]):
+		colors.append(split_line[1])
+	elif colors[0] != split_line[1] and colors[1] != split_line[1]:
+		exit("Invalid input.")
+	index = ord(split_line[0]) - ord('A')
+	if index < 0 or index > 6 or len(split_line) != 2:
+		exit("Invalid input.")
+	return index
```

Comment:
You forgot to check the first letter of the column. It should be uppercase, you are failing one of the examples!